dizoo/beergame/config/beergame_onppo_config.py

```diff
@@ -0,0 +1,79 @@
+from easydict import EasyDict
+
+beergame_ppo_config = dict(
+    exp_name='beergame_ppo_seed0',
+    env=dict(
+        collector_env_num=8,
+        evaluator_env_num=8,
+        # env_id='beergame-v2',
+        n_evaluator_episode=8,
+        stop_value=200,
+        role=0,  # 0-3 : retailer, warehouse, distributor, manufacturer
+        agent_type='bs',
+        replay_path=None
+        # type of co-player, 'bs'- base stock, 'Strm'- use Sterman formula to model typical human behavior.
+    ),
+    policy=dict(
+        cuda=True,
+        recompute_adv=True,
+        action_space='discrete',
+        model=dict(
+            obs_shape=50,  # statedim * multPerdInpt= 5 * 10
+            action_shape=5,  # the quantity relative to the arriving order
+            action_space='discrete',
+            encoder_hidden_size_list=[64, 64, 128],
+            actor_head_hidden_size=128,
+            critic_head_hidden_size=128,
+        ),
+        learn=dict(
+            epoch_per_collect=10,
+            batch_size=320,
+            learning_rate=3e-4,
+            value_weight=0.5,
+            entropy_weight=0.001,
+            clip_ratio=0.2,
```

Comment:
remove those fields that are not frequently modified, such as `clip_ratio`, `value_weight`, `unroll_len`, `grad_clip`